contents/kotlin/NullSafety.md

```diff
@@ -0,0 +1,121 @@
+# Null Safety in Kotlin
+
+Author: [Pan Haozhe](https://github.com/Haozhe321)
+
+# Overview
+Another part of this book chapter gives an introduction to Kotlin. This piece aims to discuss the null safety feature in Kotlin, and what was previously missing in Java.
```

Comment:
Comment on first sentence: Hmm actually I was under the impression that this is a book chapter in itself, i.e. you wrote a book chapter on null safety in Kotlin so I thought you meant that you'll be introducing Kotling below (but you did not). Then I thought that maybe you are referencing to another 'book chapter', but I realised that this is the first Kotlin chapter...so I am not sure if you want to even include the sentence "Another part of this book chapter gives an introduction to Kotlin" here. Perhaps it can be added in the future after such a book chapter has really been created...

Reply:
Comment on second sentence: "...what was previously missing in Java" -> I think this is a very vague statement that can leave readers wondering what you are getting at. Is there any way to state clearly what you are talking about here?

Reply:
@joanneong I was referring to the book chapter that @tshradheya is writing, introduction to Kotlin. After both are merged it would make sense to refer to his one for an overview of Kotlin. 

For the sentence on Java, perhaps I can say "...and the difference between Kotlin and Java in this aspect"?